README.md

```diff
@@ -1,15 +1,17 @@
 # PmcMiller
+[![Build Status](https://travis-ci.com/puppetlabs/pmc_miller.svg?branch=master)](https://travis-ci.com/puppetlabs/pmc_miller)
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/pmc_miller`. To experiment with that code, run `bin/console` for an interactive prompt.
 
-TODO: Delete this and the text above, and describe your gem
+PmcMiller provides an interface to read and analyze data gathered by
```

Comment:
That is far beyond the scope of SLV-730.  That functionality sounds like another gem entirely.  PmcBaker perhaps. :)

Reply:
All you would need is a writer. Should be one class similar to the reader.  And it would make more sense to add it to this gem so you can reuse existing code that would be shared.  It is certainly outside the scope of this ticket though...